bindings/python/manifold3d.cpp

```diff
@@ -116,19 +153,18 @@ PYBIND11_MODULE(manifold3d, m) {
           "Compute the convex hull enveloping a set of 3d points.")
       .def(
           "transform",
-          [](Manifold &self, py::array_t<float> &mat) {
-            auto mat_view = mat.unchecked<2>();
```

Comment:
Nice that we can remove this - I take it nanobind doesn't do any checking by default?

Reply:
it does check, the problem is that they only allows the use of ndarrays, e.g. numpy/tensorflow arrays, instead of doing explicit cast of python lists.

Reply:
That seems pretty reasonable to me.